python-flask/python-flask-example.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask, request, abort
+import base64
+import json
+import pprint
+import datetime
+app = Flask(__name__)
+
+@app.route('/', defaults={'path': ''}, methods=['POST'])
+@app.route('/<path:path>', methods=['POST'])
+def log_payload(path):
+    # Get request payload
+    if request.content_type != 'application/json':
+        # Unsupported Media Type
+        abort(415)
+
+    # Ensure that payload is a dictionary
+    payload = request.json
+    if not isinstance(payload, dict):
+        # Payload not JSON, server error
+        abort(500)
+
+    # Get message data
+    timestamp = datetime.datetime.now()
+    message = payload.get('message',{})
+    if message:
+        # Get device ID
+        id = message.get('attributes',{}).get('deviceId','')
+        # Get base64 encoded data payload
+        data = base64.b64decode(message.get('data',''))
+        print('----- Got message from {} ({}) -------'.format(id, timestamp))
+        pprint.pprint(json.loads(data))
+    else:
+        print('----- Got message that could not be parsed ({}) -----'.format(timestamp))
+        print(payload)
+    return payload
```

Comment:
A 200 response with no payload should be enough as response

Reply:
Sure, changed it to `OK` with code 200, makes it more clear that it actually works if you try the endpoint manually using Postman.